integration/test/test.ts

```diff
@@ -70,20 +72,44 @@ function log(message: string): void {
   }
 }
 
+declare type RunCallback =
+    (err: Error|null, stdout?: string, stderr?: string) => void;
+
+function trimNewline(text: string): string {
+  return text && text.endsWith('\n') ? text.slice(0, text.length - 1) : text;
```

Comment:
Could we do `text.endsWith('\n') ? ...`?

Reply:
I've updated the code to use `process.stdout.write`.